starlight.spec

```diff
@@ -0,0 +1,19 @@
+### RPM external starlight r193
+Requires: clhep gfortran
```

Comment:
Remove `gfortran`, there are no separate external for it (it comes from GCC package).